controllers/postController.js

```diff
@@ -66,31 +68,36 @@ const list = async (req, res) => {
   }
 
   const keywordId = req.params.keywordId;
-  const { includedKeyword, cursorId } = req.query;
+  const { includedKeyword, excludedKeyword, cursorId } = req.query;
+  const includedKeywordList = includedKeyword.split(",").join("|");
+  const excludedKeywordList = excludedKeyword.split(",").join("|");
   const limit = Number(req.query.limit);
   let hasNext = false;
-  let postListResult;
+  const contentFilter =
+    excludedKeywordList.length === 0
```

Comment:
```suggestion
    isEmptyString(excludedKeywordList)
```
`excludedKeywordList`가 문자열이기 때문에, 길이를 파악하는 것보단 유틸함수에 있는 `isEmptyString` 를 사용해서 확인하는 게 더 좋을 것 같아요!

Reply:
네 동의합니다, 기존 유틸을 활용하도록 수정했어요!